jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/BuildImageMojoIntegrationTest.java

```diff
@@ -68,6 +70,31 @@
   public static final TestProject defaultTargetTestProject =
       new TestProject(testPlugin, "default-target");
 
+  @ClassRule
+  public static final TestProject servlet25Project = new TestProject(testPlugin, "war_servlet25");
+
+  private static String getGcrImageReference(String imageName) {
+    String repository = "gcr.io/" + IntegrationTestingConfiguration.getGCPProject() + '/';
+    return repository + imageName + System.nanoTime();
+  }
+
+  @Nullable
+  private static String getContent(URL url) throws InterruptedException {
+    for (int i = 0; i < 40; i++) {
+      Thread.sleep(500);
+      try {
+        HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+        if (connection.getResponseCode() == HttpURLConnection.HTTP_OK) {
+          ByteArrayOutputStream out = new ByteArrayOutputStream();
+          ByteStreams.copy(connection.getInputStream(), out);
```

Comment:
Does `connection.getInputStream()` need to be closed?
I think another way to write this would be:
```
try (InputStream inputStream = connection.getInputStream()) {
  return Blobs.writeToString(Blobs.from(inputStream));
}
```

Reply:
This looks nice. :+1: